halo2/src/mock_prover.rs

```diff
@@ -106,6 +109,10 @@ mod test {
         let witness =
             executor::witgen::WitnessGenerator::new(&analyzed, degree, &fixed, query_callback)
                 .generate();
+
+        let fixed = convert_witness(fixed);
```

Comment:
to_owned_values? weird to call a function `convert_witness` on fixed columns.

Reply:
conceptually everything is witness, that's why I chose this. But will change